test/classes/nilability/nil-non-subtype.chpl

```diff
@@ -0,0 +1,26 @@
+
+class C {
+  var x : int;
+}
+
+class G { 
+  type T;
+  var x : T;
+}
+
+class D : C {
+  var y : real;
+}
+
+proc test(type A, type B) {
+  const sub = isSubtype(A, B);
+  writeln(A:string, " subtype of ", B:string, ": ", sub);
+  assert(sub != isSubtype(B, A));
+}
+
+proc main() {
+  test(C, C?);
+  test(G, G?);
+  test(G(int), G(int)?);
+  test(D, C?);
```

Comment:
Would you mind adjusting this test to also check the reverse of each of these? e.g. `test(C?, C)`?

Reply:
The 'test' function already does that, but asserts that the result is different rather than printing the result to stdout.

Reply:
Sounds good